CDC-Data-Reconciliation-Backend/server.py

```diff
@@ -75,6 +76,10 @@ def run_query(year: int):
 
 
 if __name__ == "__main__":
+    print("List of ODBC Drivers:")
+dlist = pyodbc.drivers()
+for drvr in dlist:
+    print('LIST OF DRIVERS:' + drvr)
```

Comment:
I do not believe this is necessary for the application. However, I believe this should be commented if someone would like to quickly debug the available ODBC Drivers available to them.